src/components/publishing/imageset_preview.tsx

```diff
@@ -112,31 +119,35 @@ class ImageSetPreview extends Component<Props, null> {
           </MiniInner>
         </MiniWrapper>
       )
-    } else {
-      return (
-        <FullWrapper>
-          {this.title()}
-          {this.icon()}
-        </FullWrapper>
-      )
     }
   }
   image() {
     const src = this.getImageUrl()
     const width = this.props.section.layout === "full" ? "100%" : "auto"
-    const height = this.props.section.layout === "mini" ? "100%" : "auto"
+    const height = this.props.section.layout === "full" ? "auto" : "100%"
     return <img src={src} width={width} height={height} />
   }
   title() {
-    return (
-      <TitleWrapper layout={this.props.section.layout}>
-        <Title>{this.props.section.title}</Title>
-        <SubTitle>
-          <SubTitlePrompt>View Slideshow</SubTitlePrompt>
-          <SubTitleCount>{this.props.section.images.length} Images</SubTitleCount>
-        </SubTitle>
-      </TitleWrapper>
-    )
+    if (this.props.section.title) {
+      return (
+        <TitleWrapper layout={this.props.section.layout}>
+          <Title>{this.props.section.title}</Title>
+          <SubTitle>
+            <SubTitlePrompt>View Slideshow</SubTitlePrompt>
+            <SubTitleCount>{this.props.section.images.length} Images</SubTitleCount>
+          </SubTitle>
+        </TitleWrapper>
+      )
+    } else {
+      return (
+        <TitleWrapper layout={this.props.section.layout}>
+          <Title>{this.props.section.images.length} Images</Title>
+          <SubTitle>
+            <SubTitlePrompt>View Slideshow</SubTitlePrompt>
+          </SubTitle>
+        </TitleWrapper>
+      )
+    }
   }
   icon() {
     return (
```

Comment:
Random comment not having to do with the diff: I saw some media queries in this component, but we should make use of our new `pMedia` helper [here](https://github.com/artsy/reaction/blob/master/src/components/helpers.tsx#L39)